lib/types/row.js

```diff
@@ -13,17 +37,24 @@ function Row(columns) {
  */
 Row.prototype.get = function (columnName) {
   if (typeof columnName === 'number') {
-    if (this.__columns && this.__columns[columnName]) {
-      columnName = this.__columns[columnName].name;
-    }
-    else {
-      throw new Error('Column not found');
-    }
+    return this._getValueFunc(columnName)();
```

Comment:
One possible downside of this is if someone repeatedly calls get(col) on the same column it will be evaluated every time where previously it was only done once.  It doesn't seem like a common use case, but we could memoize the evaluated value so next time the function is called it can just reuse that.  The downside of this is that the storage cost is higher.  So I kind of prefer it with this change as it is more likely get(col) will only be called at most once per column.